extensions/script/python/PythonScriptEngine.h

```diff
@@ -57,7 +57,7 @@ struct Interpreter {
 
 Interpreter *getInterpreter();
 
-class PythonScriptEngine : public script::ScriptEngine {
+class __attribute__((visibility("default"))) PythonScriptEngine : public script::ScriptEngine {
```

Comment:
This may not compile on msvc, try doing it with a platform-specific macro.

Reply:
It looks like we are not testing this extension on msvc. I'm fine with ignoring the error until we actually want to support this on windows.